### PR TITLE
Ramya/host exerciser rev5

### PR DESCRIPTION
**Pull request by ramyavex**
Checking the capability set or not for the binary.
If capability not set with non-root user then display the error. 
If capability set with non-root user then read the performance counter and display.

Added libcap-devel for capability support.

### CODE REVIEW

File: samples/host_exerciser/host_exerciser_cmd.h [hunk 1]
```diff
@@ -175,7 +181,10 @@ class host_exerciser_cmd : public test_command
 	fpgaperf::ptr_t perf(nullptr);
         if (host_exe_->perf_) {
             uid_t uid = getuid();
-            if (uid != 0) {
+            caps = cap_get_file("/usr/bin/host_exerciser");
```
Comment (tswhison) [inline samples/host_exerciser/host_exerciser_cmd.h:184]: Hi Ramya, the file may not always be under /usr/bin/.  Do we need the full path?

File: samples/host_exerciser/CMakeLists.txt [hunk 2]
```diff
@@ -30,6 +30,7 @@ opae_add_executable(TARGET host_exerciser
         opae-c
         opae-cxx-core
         afu-test
+        cap
```
Comment (tswhison) [inline samples/host_exerciser/CMakeLists.txt:33]: Before we can link with this new library, we need to implement a finder.  Please see opae-sdk/opae-libs/cmake/modules/Find*

File: opae-libs/.github/workflows/ccpp.yml [hunk 3]
```diff
@@ -22,7 +22,7 @@ jobs:
     - name: Get Packages
```
Comment (tswhison) [inline opae-libs/.github/workflows/ccpp.yml:22]: This file lives in opae-libs, so it must be updated at https://github.com/OPAE/opae-libs.git.

File: opae-libs/cmake/modules/FindCap.cmake [hunk 4]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/cmake -P
```
Comment (tswhison) [inline opae-libs/cmake/modules/FindCap.cmake:1]: This file is being added to opae-libs, so it must be added at https://github.com/OPAE/opae-libs.git.

File: opae-libs/cmake/modules/OPAE.cmake [hunk 5]
```diff
@@ -52,6 +52,7 @@ if(NOT libuuid_FOUND)
 endif()
```
Comment (tswhison) [inline opae-libs/cmake/modules/OPAE.cmake:52]: This file lives in opae-libs, so it must be updated at https://github.com/OPAE/opae-libs.git.

**Review by tswhison — CHANGES_REQUESTED** [review]

File: opae-libs/.github/workflows/ccpp.yml (hunk 3 above)
Comment (ramyavex) [reply opae-libs/.github/workflows/ccpp.yml:22]: Done the changes and pushed to opae-libs.git

File: opae-libs/cmake/modules/FindCap.cmake (hunk 4 above)
Comment (ramyavex) [reply opae-libs/cmake/modules/FindCap.cmake:1]: Done the changes and pushed to opae-libs.git

File: opae-libs/cmake/modules/OPAE.cmake (hunk 5 above)
Comment (ramyavex) [reply opae-libs/cmake/modules/OPAE.cmake:52]: Done the changes and pushed to opae-libs.git

**Comment by tswhison** [comment]
Hi Ramya, you'll need to unstage the changes to the files that were submitted to opae-libs.  Either unstage them here in this pull request or else create a new pull request without them.

**Comment by ramyavex** [comment]
creating a new pull request to resolve conflicts

**Pull request closed without merging**